src/containers/LoginScreen.js

```diff
@@ -15,19 +16,40 @@ class LoginScreen extends Component {
     this.setState(change)
   }
 
+  setErrors = errors => {
+    this.setState({ errors: errors })
+  }
+
   createAccount = () => {
-    this.fetchAccount('http://localhost:3000/create')
+    const { setAccountInfo } = this.props
+
+    this.fetchAccount('http://localhost:3000/create').then(json => {
+      if (json.errors) {
+        this.setErrors(json.errors)
+      } else {
+        setAccountInfo(json)
+        this.props.history.push('/')
+      }
+    })
   }
 
   loginAccount = () => {
-    this.fetchAccount('http://localhost:3000/login')
+    const { setAccountInfo } = this.props
+
+    this.fetchAccount('http://localhost:3000/login').then(json => {
+      if (json.errors) {
+        this.setErrors(json.errors)
+      } else {
+        setAccountInfo(json)
+        this.props.history.push('/')
+      }
+    })
   }
 
   fetchAccount = url => {
     const { email, password } = this.state
-    const { setAccountInfo } = this.props
 
-    fetch(url, {
+    return fetch(url, {
```

Comment:
cool, need you to walk through this with me - why do a post when we want to fetch account of existing user?